dask/dataframe/tests/test_optimize_dataframe.py

```diff
@@ -15,10 +15,8 @@
 
 
 def test_column_optimizations_with_bcolz_and_rewrite():
-    try:
-        import bcolz
-    except ImportError:
-        return
+    pytest.importorskip('bcolz')
```

Comment:
`importorskip` doesn't import into the namespace directly, it returns what's been imported. You need to switch this to:

```python
bcolz = pytest.importorskip('bcolz')
```

Same goes for the castra one. I find running the tests locally before committing helps catch these bugs:

```
conda install pytest    # Or pip install pytest
py.test dask/dataframe/tests/test_optimize_dataframe.py
```